resources/js/photoinit.js

```diff
@@ -3,6 +3,7 @@
 function initPhotoSwipeFromDOM (gallerySelector) {
 
     let gallery;
+    let img;

```

Comment:
I don't like global variables, because they make the code less readable.

Reply:
I don't like the code duplication but I get your point and will revert these changes. It should be improved IMHO but has nothing to do with that PR.